editor/components/multi-select-scroll-into-view/index.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * External dependencies
+ */
+import scrollIntoView from 'dom-scroll-into-view';
+
+/**
+ * WordPress dependencies
+ */
+import { Component } from '@wordpress/element';
+import { query } from '@wordpress/data';
+
+class MultiSelectScrollIntoView extends Component {
+	componentDidUpdate() {
+		// Relies on expectation that `componentDidUpdate` will only be called
+		// if value of `extentUID` changes.
+		this.scrollIntoView();
+	}
+
+	/**
+	 * Ensures that if a multi-selection exists, the extent of the selection is
+	 * visible within the viewport.
+	 *
+	 * @return {void}
+	 */
+	scrollIntoView() {
+		const { extentUID } = this.props;
+		if ( ! extentUID ) {
+			return;
+		}
+
+		const extentNode = document.querySelector( '[data-block="' + extentUID + '"]' );
+		if ( ! extentNode ) {
+			return;
+		}
+
+		scrollIntoView(
+			extentNode,
+			extentNode.closest( '.edit-post-layout__content' ),
```

Comment:
It should be easier to use a prop for this className now since it's an "external" className.

Reply:
I'm wondering if this is necessary at all...